zammad/Chart.yaml

```diff
@@ -1,7 +1,8 @@
 apiVersion: v2
 name: zammad
-version: 10.3.4
-appVersion: 6.2.0-1
+version: 10.4.0
+# appVersion: 6.3.0-0
+appVersion: develop
```

Comment:
Do you want to merge it like this?
If so, why?
After zammad/zammad gets the 6.3 tag we also have a real version to release here?

Reply:
No, this MR is only a preperation before the release, which is scheduled in a few weeks. I will merge it only after, and switch the version to the commented out one `6.3.0-0`.

I created this MR so that we can test the changes before, and are ready when the release comes. Sorry for making that not clear enough.

Reply:
Ah, ok. Thanks for clarification :)
So beside app version it looks good to me as CI runs successfull.
I'll approve as soon as the correct version is set :)